src/common/directives/autoFocus.directive.js

```diff
@@ -3,11 +3,13 @@ import angular from 'angular';
 let directiveName = 'autoFocus';
 
 /* @ngInject */
-function desigSrc() {
+function desigSrc($timeout) {
   return {
     restrict: 'A',
     link:     function ( scope, element ) {
-      element[0].focus();
+      $timeout(function () {
```

Comment:
I guess this is waiting for something else to load. Timeouts aren't ideal but as long as this consistently works it should be fine. Would $scope.$apply or $digest work here? I'm not entirely sure what we're waiting for.